module/purpose/AllRounder/src/AllRounder.cpp

```diff
@@ -164,7 +164,8 @@ namespace module::purpose {
         // Second argument is priority - higher number means higher priority
         emit<Task>(std::make_unique<FindBall>(), 1);    // if the look/walk to ball tasks are not running, find the ball
         emit<Task>(std::make_unique<LookAtBall>(), 2);  // try to track the ball
-        emit<Task>(std::make_unique<WalkToKickBall>(), 3);  // try to walk to the ball and align towards opponents goal
+        emit<Task>(std::make_unique<WalkToBall>(), 3);  // try to walk to the ball and align towards opponents goal
```

Comment:
Is `WalkToBall` needed? It is a basic linear walk to ball, whereas `WalkToKickBall` positions behind the ball towards the goal.